pkg/machine/hyperv/machine.go

```diff
@@ -633,7 +633,6 @@ func (m *HyperVMachine) startHostNetworking() (string, machine.APIForwardingStat
 	}
 
 	c := cmd.Cmd(gvproxyBinary)
-	c.ExtraFiles = []*os.File{dnr, dnw, dnw}
```

Comment:
`dnr`, `dnw`, `dnw` represent Stdin, Stdout, Stderr respectively. This is a mistake on my part. I believe this should be 
```go
c.Stdin = dnr
c.Stdout = dnw
c.Stderr = dnw
```
I mimicked how we did it with `os.ProcAttr` and made a false assumption it behaved the same way. I should have looked at the go docs where it says `c.ExtraFiles` isn't supported on windows and it also doesn't behave the same.